vtex/omnishipping
# CHK-132: Fix error in console in pickup modal

#### What is the purpose of this pull request?
Omnishipping was throwing an error in the browser console because it was trying to access a `valid` property of a `postalCode` that did not exist. This only seemed to happen in scenarios where the pickup modal did not ask for a postal code to find stores.

#### How should this be manually tested?
1. [Go to the shipping step](https://personalpy.myvtex.com/checkout/?cartman=on&orderFormId=4c080526cf76492ea7ce15d368fa0fdd&workspace=marcos#/shipping)
2. Search for a pickup point in Central > Asunción
3. There should be no errors in the browser console.

#### Types of changes
